src/parser/reduction.rs

```diff
@@ -1062,6 +1062,7 @@ pub(crate) struct ReductionMappingEntry<'a> {
 }
 
 impl<'a> ReductionMappingEntry<'a> {
+    #[allow(dead_code)]
     pub(crate) fn assumption(&self) -> &PackageInstanceName<'a> {
         &self.assumption
     }
```

Comment:
Maybe let's just remove the getter? We can trivially re-add it if we need it at some point.